tests/mysql/model/test_model.py

```diff
@@ -126,6 +143,45 @@ def test_serialize_with_hidden(self):
             self.assertTrue(profile.serialize().get("id"))
             self.assertFalse(profile.serialize().get("password"))
 
+        def test_serialize_with_visible(self):
+            profile = ProfileSerializeWithVisible.hydrate(
+                {
+                    "name": "Joe",
+                    "id": 1,
+                    "password": "secret",
+                    "email": "joe@masonite.com",
+                }
+            )
+            self.assertTrue(
+                {"name": "Joe", "email": "joe@masonite.com"}, profile.serialize()
+            )
+
+        def test_serialize_with_visible_and_initial_data(self):
```

Comment:
so we can just remove this test and remove that serialized_dictionary in the parameter list and set serialized_dictionary to be the model attributes